contracts/src/starknet/rewaave/token_bridge.cairo

```diff
@@ -179,3 +184,19 @@ func handle_deposit{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_chec
 
     return ()
 end
+
+@external
+func mint_rewards{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+        recipient : felt, amount : Uint256):
+    # get the address of the ETHStaticAToken
+    let (l2_token) = get_caller_address()
+    # Verify that it's a valid token by checking for its counterpart on l1
+    let (l1_token) = l2_token_to_l1_token.read(l2_token)
+    with_attr error_message("L1 token {l1_token} not found"):
+        assert_not_zero(l1_token)
+    end
+    # mints rewAAVE for user
+    IL2Token.mint(rewAAVE_token, recipient, amount)
```

Comment:
Please use IERC20 impl

Reply:
should be implemented  for deposit & withdrawal as well 👍🏻